src/Core/Grid/Definition/Factory/OrderGridDefinitionFactory.php

```diff
@@ -211,6 +213,7 @@ protected function getColumns()
                 ->setOptions([
                     'field' => 'total_paid_tax_incl',
                     'is_paid_field' => 'paid',
+                    'clickable' => true,
```

Comment:
Mmm, I don't get it a `LinkColumn` shouldn't be clickable.. unless this link matches the row link But still it seems weird

Reply:
Did you misread ? 😅 `LinkColumn` for customer name is not clickable, here it's `OrderPriceColumn` that is made clickable